### DISCUSSION TITLE
Next/Image not found?

### DISCUSSION THREAD

**Discussion by helloimalastair** [vercel/next.js#18882] [Help] [answered]
Can I use the Image component from a separate folder? When I use it in the Components folder, it says "Module not found: Can't resolve 'next/image'", and I can't figure out if it is the wrong folder, or something else.

**Comment by jamesmosier** [comment, marked as answer]
You can use it in any folder within your Next.js project. What version are you running of Next?

**Reply by helloimalastair** [reply]
Thank You! Sorry I hadn't updated this earlier, but I found out that my problem was that I had not updated Next in a while, and so it didn't work as expected. It works fine now!

**Comment by hpachy** [comment]
Hello I got the same problem since 1 day... I'm sharing with you my json package to figured out what's going on

```json
{
  "name": "showcase-front",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start"
  },
  "dependencies": {
    "next": "10.0.7",
    "react": "17.0.1",
    "react-device-detect": "^1.15.0",
    "react-dom": "17.0.1",
    "react-modal": "^3.12.1",
    "react-use": "^17.1.1"
  },
  "devDependencies": {
    "@types/node": "^14.14.31",
    "@types/react": "^17.0.2",
    "typescript": "^4.1.5"
  }
}
```
$ next build
>Browserslist: caniuse-lite is outdated. Please run:
npx browserslist@latest --update-db
info  - Creating an optimized production build  
Failed to compile.

ModuleNotFoundError: Module not found: Error: Can't resolve 'next/image' in '/Users/me/***/***/components/HomePage'


> Build error occurred
Error: > Build failed because of webpack errors
    at build (/usr/local/lib/node_modules/next/dist/build/index.js:15:918)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:93:5)

**Reply by helloimalastair** [reply]
No guarantees on this, but you can try reloading all your modules. Start by removing all of your modules from your `package.json` file, like so:
```
{
  "name": "showcase-front",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start"
  },
  "dependencies": {},
  "devDependencies": {}
}
```
Next, delete your `node_modules` folder, using either a folder explorer or command. Lastly, reinstall all of your modules using this command: `npm install next react react-device-detect react-dom react-modal react-use`, and for your dev dependencies, run  
`npm install -D @types/node @types/react typescript`. This assumes you have not added any further modules since you posted your `package.json` file. If this works, you should be good to go!

**Reply by hpachy** [reply]
it's Not fixing my problem... I think it's because I got svgs names changed the case on my git... That's the origin of my problem

**Reply by zooeyking** [reply]
![image](https://github.com/vercel/next.js/assets/16650705/e82b7ecb-402c-499c-a25c-716a4d62c871)

you can try this.

**Comment by hamed-iam** [comment]
Unfortunately I had the same issue and I couldnt fix it, and I was running the latest next 10.0.9, react 17.0.2 I removed the modules and .next and reinstall them all and still got the same error " Module not found: Can't resolve 'next/images' ". I had to go manual and not use this feature .

**Reply by hpachy** [reply]
My problem was because I changed some of my files names (only the case, ex: EXEMPLE.tsx => Exemple.tsx) and github not recognize this kind of changes

you maybe have the same problem as me

**Reply by kanlancb** [reply]
@hpachy How did you fix it?

**Reply by hpachy** [reply]
because github doesn't case about camelCase : 
- you must change the file name "exemple.js" to "ExempleBis.js"
- you must push changes
- to finish rename your file "ExempleBis.js" to "Exemple.js" and repush

Not the best technic but it's work

**Comment by abhinavakhil** [comment]
Solution that worked for me is by updating nextjs to the latest version. try this ->  npm install next@latest

**Reply by PolarCoding85** [reply]
This fixed my issue! Thanks.

**Reply by abhinavakhil** [reply]
glad it helped fix your problem.

**Comment by lukaskf** [comment]
Had this same issue was very confused since everything ran locally but failed on deployment.
The issue was I capitalized `Image`.

Wrong:
`import Image from "next/Image"`

Right:
`import Image from "next/image"`

**Reply by jakepow07** [reply]
This worked for me. Thanks!

**Comment by danielangelo** [comment]
If somebody still has this issue and none of the above worked try this:


Open `tsconfig.json` and add `"moduleResolution": "node"` inside `compilerOptions`

**Reply by davdleet** [reply]
this worked for me thanks!

**Comment by lolmaus** [comment]
I've had this problem and literally nothing helped.

I'm on Windows, and the code is stored and running in WSL. I opened the codebase in VS Code by running `code .` inside WSL. For some reason, it defaulted to opening the code using the `\\wsl.localhost\Ubuntu` SMB network share path.

I re-opened the codebase in VS Code using the WSL Remote, and missing module errors are gone!
